consensus/types/src/config_and_preset.rs

```diff
@@ -99,34 +102,34 @@ pub fn get_extra_fields(spec: &ChainSpec) -> HashMap<String, Value> {
     let u32_hex = |v: u32| hex_string(&v.to_le_bytes());
     let u8_hex = |v: u8| hex_string(&v.to_le_bytes());
     hashmap! {
-        "bls_withdrawal_prefix".to_uppercase() => u8_hex(spec.bls_withdrawal_prefix_byte),
-        "eth1_address_withdrawal_prefix".to_uppercase() => u8_hex(spec.eth1_address_withdrawal_prefix_byte),
-        "domain_beacon_proposer".to_uppercase() => u32_hex(spec.domain_beacon_proposer),
-        "domain_beacon_attester".to_uppercase() => u32_hex(spec.domain_beacon_attester),
-        "domain_randao".to_uppercase()=> u32_hex(spec.domain_randao),
-        "domain_deposit".to_uppercase()=> u32_hex(spec.domain_deposit),
-        "domain_voluntary_exit".to_uppercase() => u32_hex(spec.domain_voluntary_exit),
-        "domain_selection_proof".to_uppercase() => u32_hex(spec.domain_selection_proof),
-        "domain_aggregate_and_proof".to_uppercase() => u32_hex(spec.domain_aggregate_and_proof),
-        "domain_application_mask".to_uppercase()=> u32_hex(spec.domain_application_mask),
+        "bls_withdrawal_prefix".to_uppercase() => u8_hex(BLS_WITHDRAWAL_PREFIX),
+        "eth1_address_withdrawal_prefix".to_uppercase() => u8_hex(ETH1_ADDRESS_WITHDRAWAL_PREFIX),
+        "domain_beacon_proposer".to_uppercase() => u32_hex(DOMAIN_BEACON_PROPOSER),
+        "domain_beacon_attester".to_uppercase() => u32_hex(DOMAIN_BEACON_ATTESTER),
+        "domain_randao".to_uppercase()=> u32_hex(DOMAIN_RANDAO),
+        "domain_deposit".to_uppercase()=> u32_hex(DOMAIN_DEPOSIT),
+        "domain_voluntary_exit".to_uppercase() => u32_hex(DOMAIN_VOLUNTARY_EXIT),
+        "domain_selection_proof".to_uppercase() => u32_hex(DOMAIN_SELECTION_PROOF),
+        "domain_aggregate_and_proof".to_uppercase() => u32_hex(DOMAIN_AGGREGATE_AND_PROOF),
+        "domain_application_mask".to_uppercase()=> u32_hex(APPLICATION_DOMAIN_BUILDER),
```

Comment:
I think we should use 2 separate constants for the mask vs the builder domain even though `DOMAIN_APPLICATION_MASK == APPLICATION_DOMAIN_BUILDER`. probably better to make a specific `DOMAIN_APPLICATION_MASK` constant available